addons/sale_timesheet/models/project.py

```diff
@@ -183,43 +182,6 @@ def _compute_sale_line_id(self):
             ], limit=1)
             project.sale_line_id = sol or project.sale_line_employee_ids.sale_line_id[:1]  # get the first SOL containing in the employee mappings if no sol found in the search
 
-    @api.depends('sale_line_id.product_uom_qty', 'sale_line_id.product_uom')
-    def _compute_allocated_hours(self):
-        sol_ids = self._fetch_sale_order_item_ids()
```

Comment:
Okay but if the allocated_hours is not set, what do we do? 🤔

Reply:
I asked to @aju-odoo and indeed we just want to compute the `allocated_hours` when the project is generated by a SO. 

So, it is okay to remove the method. However, I wonder if the fix should be done in 16 (can you check with AJU?). If the fix has to be done in 16.0 then we cannot remove the method, we have to deprecate it since we are in a stable version and in master, we will remove it.